openquake/commonlib/tests/source_test.py

```diff
@@ -34,13 +34,15 @@
 from openquake.commonlib.source import parse_source_model, DuplicatedID
 from openquake.commonlib.nrml import nodefactory
 from openquake.commonlib.node import read_nodes
-from openquake.baselib.general import assert_close
+from openquake.baselib.general import assert_close, assert_close_seq
 
```

Comment:
`assert_close_seq` is really an internal function. You should just use `assert_close`.